xDrip/Scenes/InitialSetup/ViewControllers/G6/InitialSetupG6ConnectViewController/InitialSetupG6ConnectViewController.swift

```diff
@@ -28,16 +42,16 @@ final class InitialSetupG6ConnectViewController: InitialSetupAbstractStepViewCon
         interactor?.doCompleteCustomDeviceStep(request: request)
     }
     
-    private let worker = InitialSetupDexcomG6ConnectionWorker()
+    private var worker: InitialSetupDexcomG6ConnectionWorkerProtocol?
```

Comment:
Why `var` and optional? Make it `private let worker: InitialSetupDexcomG6ConnectionWorkerProtocol` and modify init to:
```swift
required init(connectionWorker: InitialSetupDexcomG6ConnectionWorkerProtocol) {
         worker = connectionWorker
         super.init()
     }
```